src/Theme/ClassicTheme.php

```diff
@@ -46,15 +46,20 @@ public function onTestFinished(TestResult $result): void
             $throwable = $throwable->previous();
         }
 
-        if ($result->trace && !$result->trace->suppressed) {
-            printf(
-                Color::colorize("fg-$statusColour", '%s%s: %s in %s on line %s%1$s%1$s'),
-                PHP_EOL,
-                $result->status->name,
-                $result->trace->message,
-                $result->trace->file,
-                $result->trace->line,
-            );
+        $lastKey = array_key_last($result->uniqueTraces);
```

Comment:
Considering `foreach` variables are not scoped to the `foreach` in PHP (they leak to the parent scope), this isn't really necessary. That is, we can just write:

```php
if (isset($trace)) {
    echo PHP_EOL;
}
```

This `if` condition tells us if the loop looped or not.

Reply:
This part of the code is wrong anyway, either this or the first version I wrote. I need to review it entirely.

Reply:
I've restored and fixed my original change, which uses a boolean to track first _visible_ issue.

I think that what was driving me crazy was the fact that there're either 2 or 3 blank lines before the summary depending on whether the last test had issues. That's the existing behaviour and my changes keep it. Why? Because I didn't want to bulk rewrite existing *.phpt files.